.travis.yml

```diff
@@ -39,12 +39,16 @@ env:
   - PYTHONASYNCIODEBUG=x
   - PYTHONASYNCIODEBUG=
 
+matrix:
+  allow_failures:
+    - python: "nightly"
```

Comment:
the established indentation style across this file doesn't require extra space in sequences.